scripts/player.gd

```diff
@@ -9,16 +13,31 @@ extends CharacterBody2D
 @export_range(0, 1000, 10, "suffix:px/s") var speed: float = 500.0:
 	set = _set_speed
 
+## How fast does your character accelerate?
+@export_range(0, 5000, 1000, "suffix:px/s²") var acceleration: float = 5000.0
+
 ## How high does your character jump? Note that the gravity will
 ## be influenced by the [member GameLogic.gravity].
 @export_range(-1000, 1000, 10, "suffix:px/s") var jump_velocity = -880.0
 
+## Can your character jump a second time while still in the air?
+@export var double_jump: bool = false
+
+# If positive, the player is either on the ground, or left the ground less than this long ago
+var coyote_timer: float = 0
```

Comment:
Can coyote timer be turned on/off or configured from the Inspector? See https://gmtk.itch.io/platformer-toolkit for reference.

Reply:
I didn't make coyote time or jump buffering configurable in the inspector because they're quite non-obvious parameters with a very narrow range of sensible settings. But maybe it's OK to expose them anyway with a good tooltip, maybe in an "advanced" group.

Reply:
I made these configurable, as a slider whose steps are frame durations @ 60 fps, with `0` turning them off.